rewrite-yaml/src/main/java/org/openrewrite/yaml/tree/Yaml.java

```diff
@@ -243,6 +244,22 @@ public enum Style {
             PLAIN
         }
 
+        @Deprecated
+        public Scalar(UUID id, String prefix, Markers markers, Style style, @Nullable Anchor anchor, String value) {
+            this(id, prefix, markers, style, anchor, null, value);
+        }
+
+        @JsonCreator
```

Comment:
You could instead also have told Lombok to add this annotation to the constructor it generates.

Reply:
Thanks! I'd considered that, but figured it'd be harder to clear out automatically as opposed to an effective all args constructor that we can remove when it's the only one left as part of Lombok best practices.